envFiles/.env.production

```
@@ -106,8 +106,17 @@ FRONTEND_URL="http://localhost:4321"
 
 
 ##### OTEL and Observability #####
-API_OTEL_ENABLED=true
-API_OTEL_SERVICE_NAME="talawa-api"
-API_OTEL_ENVIRONMENT="production"
-API_OTEL_SAMPLING_RATIO="1"
-API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces"#TODO: after integrating Observability tool
\ No newline at end of file
+API_OTEL_ENABLED=false
+API_OTEL_ENVIRONMENT=production
+API_OTEL_EXPORTER_OTLP_ENDPOINT="http://localhost:4318/v1/traces" # TODO: confirm this is the correct OTLP exporter endpoint for your observability tool
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)